static/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html data-bs-theme="dark" lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta content="width=device-width, initial-scale=1" name="viewport">
+		<title>ankerctl</title>
+		<link crossorigin="anonymous" href="//cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" rel="stylesheet">
+	</head>
+	<body>
+		<div class="container my-5">
+			<div class="p-2 text-center bg-body-tertiary rounded-3">
+				<h1 class="text-body-emphasis display-3">ankerctl</h1>
+				<p class="lead">Congratulations on running ankerctl&hellip;</p>
```

Comment:
It feels a bit odd to have "..." at the end of a headline. But an exclamation mark would also be too much, I think. Maybe we just remove `&hellip;`?